docs/models-definition.md

```diff
@@ -72,6 +72,37 @@ const Foo = sequelize.define('foo', {
 
 The comment option can also be used on a table, see [model configuration][0]
 
+## Timestamps
+
+By default, Sequelize will add the attributes `createdAt` and `updatedAt` to your model so you will be able to know when the database entry went into the db and when it was updated last. 
+
+Note that if you are using Sequelize migrations you will need to add the `createdAt` and `updatedAt` fields to your migration definition:
+
+```js
+module.exports = {
+  up(queryInterface, Sequelize) {
+    return queryInterface.createTable('my-table', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+
+      // Timestamps
+      createdAt: Sequelize.DATE,
+      updatedAt: Sequelize.DATE,
+    })
+  },
+  down(queryInterface, Sequelize) {
+    return queryInterface.dropAllTables()
```

Comment:
This was taken from my initial DB migration. Could change to dropping created table